src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import Page from './Page';
```

Comment:
```suggestion
import React, { useState } from 'react';

import Page from './Page';
```
한 줄 공백을 주시면 좋을 것 같습니다! 지금은 모듈이 별로 없지만, 모듈 간에도 역할이 있고, 분리를 해두는 습관을 두면 좋을 것 같아요 💯 
다른 파일들도 적용해주세요~

Reply:
변경 완료했습니다. 이제부터 습관을 잘 들여놓고 얼른 긴코드를 다룰만큼 실력이 늘어서 이 습관이 뿌듯해지는 순간이 왔으면 좋겠습니다 감사합니다! :):)